var/spack/repos/builtin/packages/py-asn1crypto/package.py

```diff
@@ -0,0 +1,37 @@
+##############################################################################
```

Comment:
It looks like there's an `Error:` file above this that was accidentally committed.